packages/hash/frontend/src/pages/[account-slug]/types/entity-type/custom-property-type-menu.tsx

```diff
@@ -0,0 +1,221 @@
+          variant="smallTextLabels"
+          sx={{ paddingTop: 1.25, color: ({ palette }) => palette.gray[70] }}
+        >
+          Advanced users can specify property objects as well as arrays of data
+          types and/or property objects as expected values.
+        </Typography>
+      </Stack>
+
+      <Stack
+        gap={3}
+        sx={({ palette }) => ({
+          maxHeight: "40vh",
+          overflow: "auto",
+          paddingY: 2.25,
+          paddingX: 1.5,
+          background: palette.gray[20],
+          border: `1px solid ${palette.gray[30]}`,
+          ...(!creatingPropertyId
+            ? {
+                borderBottomRightRadius: 4,
+                borderBottomLeftRadius: 4,
+              }
+            : { borderBottomWidth: 0 }),
+        })}
+      >
+        {!creatingPropertyId ? (
+          <>
+            <Stack direction="row" gap={1.75}>
+              <Button
+                size="small"
+                variant="tertiary"
+                endIcon={<FontAwesomeIcon icon={{ icon: faCube }} />}
+              >
+                Create a property object
+              </Button>
+              <Stack gap={0.25}>
+                <Typography
+                  variant="smallTextLabels"
+                  sx={{
+                    fontSize: 11,
+                    fontWeight: 500,
+                    color: ({ palette }) => palette.gray[60],
+                  }}
+                >
+                  CONTAINS
+                </Typography>
+                <CustomChip
+                  color="purple"
+                  label="PROPERTY TYPES"
+                  borderColor="#FFF"
+                />
+              </Stack>
+            </Stack>
+
+            <Stack direction="row" gap={1.75}>
+              <Button
+                size="small"
+                variant="tertiary"
+                endIcon={<FontAwesomeIcon icon={faList} />}
+                onClick={() => {
+                  const id = uniqueId();
+
+                  setValue("creatingPropertyId", id);
+                  setValue("flattenedPropertyList", {
+                    [id]: {
+                      id,
+                      data: getDefaultData("array"),
+                    },
+                  });
+                }}
+              >
+                Create an array
+              </Button>
+              <Stack gap={0.25}>
+                <Typography
+                  variant="smallTextLabels"
+                  sx={{
+                    fontSize: 11,
+                    fontWeight: 500,
+                    color: ({ palette }) => palette.gray[60],
+                  }}
+                >
+                  ALLOWS COMBINING
+                </Typography>
+                <Stack direction="row" gap={1}>
+                  <CustomChip color="purple" label="PROPERTY OBJECTS" />
+                  <CustomChip color="blue" label="DATA TYPES" />
+                  <CustomChip color="navy" label="ARRAY" />
```

Comment:
```suggestion
                  <CustomChip color="navy" label="ARRAYS" />
```

Reply:
Fixed in 143ecff2f511740c90060e7e4a5c7d6c3e24a0a0. Well spotted